.jshintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is it possible to put a comment with a link pointing to where these flags are defined?